src/test/regress/expected/fast_path_router_modify.out

```diff
@@ -207,19 +207,19 @@ ERROR:  could not run distributed query with FOR UPDATE/SHARE commands
 HINT:  Consider using an equality filter on the distributed table's partition column.
 -- very simple queries on reference tables goes through fast-path planning
 DELETE FROM modify_fast_path_reference WHERE key = 1;
-DEBUG:  Distributed planning for a fast-path router query
+DEBUG:  Deferred pruning for a fast-path router query
```

Comment:
For example, we defer pruning here just because it is a reference table and we cannot find a distribution key value.

There are some other examples for deferring pruning just because the dist key column type doesn't match the const type such as int vs bigint.

So, I think I'll go ahead and defer pruning only when there is a parameter in the dist key